.jenkins/build_container.JenkinsFile

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/groovy
+def LABEL = "build-jenkins-casc-" + UUID.randomUUID().toString().take(8)
+pipeline {
+    parameters {
+        choice(name: 'TAG', choices: ['git-commit','git-tag'],  description: 'Which info to use in container tag?')
+    }
+    options {
+      timestamps()
+      ansiColor('xterm')
+    }
+    agent {
+      kubernetes {
+        label LABEL
+        yamlFile '.jenkins/pods/build_container.yaml'
+      }
+    }
+    stages {
+      stage('Container') {
+        steps {
+          if (params.TAG != '') {
+            script {
+                tag=get_tag()
+            }
```

Comment:
```suggestion
```

I fear the variable declaration will not work because it's not declared in the global scope (outside of `pipeline`). I suggest to directly call the method at the place where you need it.

Reply:
I think it works, that was a template from https://github.com/EMnify/emnify-connectivity/blob/45d76f98a1f27456cf665c1d93744aed6aae321e/.jenkins/build_container.Jenkinsfile#L22.
But anyways, I liked it, thanks :D